app/Main.hs

```diff
@@ -97,11 +99,21 @@ main = do
       r <- try (fetchTeamKeys gitHubToken orgName teamName)
       case r of
         Left err -> do
-          print (err :: SomeException)
-          exitFailure
+          let f x = if isDoesNotExistError x then Just x else Nothing
+          ecache <- tryJust f (B.readFile configCacheFile)
+          case ecache of
+            Left readErr -> do
+              print (err :: SomeException)
+              print readErr
```

Comment:
@mrkkrp I think we need to redirect this output to `stderr` instead of `stdout` since it is an error. So maybe `hPrint stderr` instead of `print`?